tests/robot-tests/tests/libs/utilities.py

```diff
@@ -263,8 +266,9 @@ def user_checks_indicator_checkbox_is_selected(indicator_label):
 
 
 def user_checks_subheaded_indicator_checkbox_is_selected(subheading_label, indicator_label):
-    sl.checkbox_should_be_selected(
-        f'xpath://*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input')
+    checkbox_selector = f'xpath://*[@id="filtersForm-indicators"]//legend[text()="{subheading_label}"]/..//label[text()="{indicator_label}"]/../input'
```

Comment:
Couldn't all of these just be in `common.robot`?

The only complicated part is the selector, and even that can be assigned to a variable in Robot Framework.

Also, as this isn't being wrapped with the try/except, it wouldn't be great from a debug standpoint.

Reply:
I don't think the try/catch blocks are required if you're calling selenium library functions directly - I'm assuming they handle that themselves.

Reply:
ah - I see  there is one webdriver call - I'll wrap it.